### PR TITLE
[4.0] convert to prepared statements Viewlevels table

### PR DESCRIPTION
**Pull request by alikon**
### Summary of Changes
converted sql to prepared statements

### Testing Instructions

-  create viewlevels
-  delete viewlevels
-  modify viewlevels
-  view  viewlevels

### Expected result
works as before

### Actual result
N/A

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c8c07c65c7119c5c871204d680858e21769f3830<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28489">issues.joomla.org/tracker/joomla-cms/28489</a>.</sub>

File: libraries/src/Table/ViewLevel.php [hunk 1]
```diff
@@ -84,13 +85,17 @@ public function check()
 			return false;
 		}
 
+		$id = (int) $this->id;
+
 		// Check for a duplicate title.
 		$db = $this->_db;
 		$query = $db->getQuery(true)
 			->select('COUNT(title)')
```
Comment (HLeithner) [inline libraries/src/Table/ViewLevel.php]: Shouldn't the title quoted too?

Comment (alikon) [reply libraries/src/Table/ViewLevel.php]: sure it need to be quoted as well

**Comment by alikon** [comment]
the reason why drone still complains is above of my skills, i give up

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**